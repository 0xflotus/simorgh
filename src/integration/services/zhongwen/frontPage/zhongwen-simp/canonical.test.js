/**
 * WARNING: DO NOT EDIT. This file is automatically generated by src/integration/utils/runTests/writeTestFiles/writeTestFile.js
 * @service zhongwen
 * @pathname /zhongwen/simp
 */

import runCanonicalTests from '../../../../pages/frontPage/canonicalTests';

describe('Canonical zhongwen frontPage', runCanonicalTests);
