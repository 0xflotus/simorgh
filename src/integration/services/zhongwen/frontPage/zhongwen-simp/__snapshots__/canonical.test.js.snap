// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Canonical zhongwen frontPage A11y I can see the skip to content link 1`] = `"跳过此内容"`;

exports[`Canonical zhongwen frontPage Analytics ATI 1`] = `"<img height=\\"1px\\" width=\\"1px\\" alt=\\"\\" style=\\"position:absolute\\" src=\\"https://logws1363.ati-host.net?s=598343&amp;s2=38&amp;p=zhongwensimp.page&amp;x1=[urn:bbc:cps:426d2030-e2dc-5202-e050-380a4e80578d]&amp;x2=[responsive]&amp;x3=[news-zhongwen]&amp;x4=[zh-Hans]&amp;x7=[index-home]&amp;x8=[simorgh-nojs]&amp;x9=[主页+-+BBC+News+中文]&amp;x11=[2016-11-29T14:34:24.000Z]&amp;x12=[2019-09-08T18:37:27.000Z]\\"/>"`;

exports[`Canonical zhongwen frontPage Footer I can click on the BBC branding and it would take me to the homepage 1`] = `"/zhongwen"`;

exports[`Canonical zhongwen frontPage Footer I can see the footer copyright and external linking text 1`] = `"© 2020 BBC. BBC对外部网站内容不负责任。 阅读了解我们对待外部链接的做法。"`;

exports[`Canonical zhongwen frontPage Header I can see the branding 1`] = `"BBC News, 中文"`;

exports[`Canonical zhongwen frontPage I can see Index Alsos 1`] = `"更多相关内容"`;

exports[`Canonical zhongwen frontPage I can see sections 1`] = `"Top story - Zhongwen"`;

exports[`Canonical zhongwen frontPage I can see sections 2`] = `"必看"`;

exports[`Canonical zhongwen frontPage I can see sections 3`] = `"新闻时事 趋势动态"`;

exports[`Canonical zhongwen frontPage I can see sections 4`] = `"知识资讯 观点角度"`;

exports[`Canonical zhongwen frontPage I can see sections 5`] = `"播客精选浏览全部"`;

exports[`Canonical zhongwen frontPage I can see sections 6`] = `"BBC学院浏览全部"`;

exports[`Canonical zhongwen frontPage I can see sections 7`] = `"浏览协助"`;

exports[`Canonical zhongwen frontPage I can see the headline 1`] = `"BBC News, 中文 - 主页"`;

exports[`Canonical zhongwen frontPage SEO Canonical link 1`] = `"http://localhost:7080/zhongwen/simp"`;

exports[`Canonical zhongwen frontPage SEO Dir attribute 1`] = `"ltr"`;

exports[`Canonical zhongwen frontPage SEO FB admins 1`] = `"100004154058350"`;

exports[`Canonical zhongwen frontPage SEO FB app ID 1`] = `"1609039196070050"`;

exports[`Canonical zhongwen frontPage SEO Heading level 1 1`] = `"BBC News, 中文 - 主页"`;

exports[`Canonical zhongwen frontPage SEO Lang attribute 1`] = `"zh-Hans"`;

exports[`Canonical zhongwen frontPage SEO Linked data 1`] = `
Object {
  "@context": "http://schema.org",
  "@type": "WebPage",
  "image": Object {
    "@type": "ImageObject",
    "height": 576,
    "url": "https://news.files.bbci.co.uk/ws/img/logos/og/zhongwen.png",
    "width": 1024,
  },
  "mainEntityOfPage": Object {
    "@id": "http://localhost:7080/zhongwen/simp",
    "@type": "WebPage",
    "name": "主页",
  },
  "publisher": Object {
    "@type": "NewsMediaOrganization",
    "logo": Object {
      "@type": "ImageObject",
      "height": 576,
      "url": "https://news.files.bbci.co.uk/ws/img/logos/og/zhongwen.png",
      "width": 1024,
    },
    "name": "BBC News 中文",
    "publishingPrinciples": "https://www.bbc.com/zhongwen/simp/institutional-51359584",
  },
  "thumbnailUrl": "https://news.files.bbci.co.uk/ws/img/logos/og/zhongwen.png",
  "url": "http://localhost:7080/zhongwen/simp",
}
`;

exports[`Canonical zhongwen frontPage SEO OG description 1`] = `"BBC中文网是面向全球华人并每日更新的新闻资讯网站，提供客观及时和准确的内容以及对新闻故事的深入分析和评论。我们也提供繁体版脸书。"`;

exports[`Canonical zhongwen frontPage SEO OG image 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/zhongwen.png"`;

exports[`Canonical zhongwen frontPage SEO OG image alt 1`] = `"BBC News 中文"`;

exports[`Canonical zhongwen frontPage SEO OG locale 1`] = `"zh-hans"`;

exports[`Canonical zhongwen frontPage SEO OG site name 1`] = `"BBC News 中文"`;

exports[`Canonical zhongwen frontPage SEO OG title 1`] = `"主页 - BBC News 中文"`;

exports[`Canonical zhongwen frontPage SEO OG type 1`] = `"website"`;

exports[`Canonical zhongwen frontPage SEO OG url 1`] = `"http://localhost:7080/zhongwen/simp"`;

exports[`Canonical zhongwen frontPage SEO Page title 1`] = `"主页 - BBC News 中文"`;

exports[`Canonical zhongwen frontPage SEO Robots meta tag 1`] = `"noodp,noydir"`;

exports[`Canonical zhongwen frontPage SEO Twitter card 1`] = `"summary_large_image"`;

exports[`Canonical zhongwen frontPage SEO Twitter creator 1`] = `"@bbcchinese"`;

exports[`Canonical zhongwen frontPage SEO Twitter description 1`] = `"BBC中文网是面向全球华人并每日更新的新闻资讯网站，提供客观及时和准确的内容以及对新闻故事的深入分析和评论。我们也提供繁体版脸书。"`;

exports[`Canonical zhongwen frontPage SEO Twitter image alt 1`] = `"BBC News 中文"`;

exports[`Canonical zhongwen frontPage SEO Twitter image src 1`] = `"https://news.files.bbci.co.uk/ws/img/logos/og/zhongwen.png"`;

exports[`Canonical zhongwen frontPage SEO Twitter site 1`] = `"@bbcchinese"`;

exports[`Canonical zhongwen frontPage SEO Twitter title 1`] = `"主页 - BBC News 中文"`;

exports[`Canonical zhongwen frontPage Story Promo I can see a headline 1`] = `"SIMP - 香港示威者为何希望美国会通过人权民主法案"`;

exports[`Canonical zhongwen frontPage Story Promo I can see an image 1`] = `
<img
  alt="示威者星期日下午开始在中环地铁站口纵火，地铁站两旁是中环名牌商业购物区。"
  class="StyledImg-sc-7vx2mr-0 hIxkbt"
  sizes="(max-width: 600px) 100vw, (max-width: 1008px) 50vw, 496px"
  src="https://ichef.bbci.co.uk/news/660/cpsprodpb/5195/production/_108658802_hi056347143_01.jpg"
  srcset="https://ichef.bbci.co.uk/news/70/cpsprodpb/5195/production/_108658802_hi056347143_01.jpg 70w, https://ichef.bbci.co.uk/news/95/cpsprodpb/5195/production/_108658802_hi056347143_01.jpg 95w, https://ichef.bbci.co.uk/news/144/cpsprodpb/5195/production/_108658802_hi056347143_01.jpg 144w, https://ichef.bbci.co.uk/news/183/cpsprodpb/5195/production/_108658802_hi056347143_01.jpg 183w, https://ichef.bbci.co.uk/news/240/cpsprodpb/5195/production/_108658802_hi056347143_01.jpg 240w, https://ichef.bbci.co.uk/news/320/cpsprodpb/5195/production/_108658802_hi056347143_01.jpg 320w, https://ichef.bbci.co.uk/news/660/cpsprodpb/5195/production/_108658802_hi056347143_01.jpg 660w"
  width="1024"
/>
`;
